package.json

```diff
@@ -19,7 +19,7 @@
     "doc": "node ./scripts/doc/build.js",
     "prettier": "prettier --write '*.js' 'src/**/*.js'",
     "lint:js": "eslint '*.js' 'scripts/**/*.js' 'src/**/*.js'",
-    "lint:html": "node ./scripts/lint-html.js 'src/**/*.html'",
+    "lint:html": "htmlhint src/**/*.html",
```

Comment:
The quotes around the glob might be significant because it will use node glob vs local shell glob

Reply:
Interesting, I didn't know this detail. This [article](https://medium.com/@jakubsynowiec/you-should-always-quote-your-globs-in-npm-scripts-621887a2a784) seems to make a good case that I should be using Node glob if I want to aim for cross-platform compatibility, so I'll do that.